documentation/modules/ROOT/pages/transformations/vitess-use-local-vgtid.adoc

```diff
@@ -0,0 +1,82 @@
+:page-aliases: configuration/vitess-use-local-vgtid.adoc
+[id="vitess-use-local-vgtid"]
+= Vitess Use Local VGTID
+
+:toc:
+:toc-placement: macro
+:linkattrs:
+:icons: font
+:source-highlighter: highlight.js
+
+toc::[]
+
+The `UseLocalVgtid` SMT converts the value of the VGTID in the event record from a list of VGTIDs for several shards to the VGTID for the single shard that received the change for this event.
+This transformation is designed for use only with the {prodname} connector for Vitess.
+
+The SMT extracts the `shard` field in the `source` block of the transaction boundary event record to determine the shard that received the change. 
+The SMT then sets that value of the VGTID in the even record to the value of the VGTID for the specified shard.
+Other VGTID values are removed from the event record.
+The following example shows how to add the SMT to the connector configuration so that it converts the list of VGTIDs to the value of the local VGTID.
```

Comment:
```suggestion
The following example shows how to add the SMT to the connector configuration so that it converts lists of the VGTIDs for multiple shards to the value of a single local VGTID.
```